server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/PublishBallotsToAudit.java

```diff
@@ -1,132 +0,0 @@
-/*
```

Comment:
We're still using this endpoint in `master`, so we should change that before we merge this commit.

Reply:
Yup

Reply:
Okay, as of the latest `sprint` branch, we no longer use this endpoint. We're safe to merge after we merge #559.

Reply:
#559 has landed, so this deletion will no longer break anything.